pkg/genevalogging/genevalogging.go

```diff
@@ -257,6 +259,16 @@ func (g *genevaLogging) CreateOrUpdate(ctx context.Context) error {
 		return err
 	}
 
+	timeoutCtx, cancel := context.WithTimeout(ctx, 30*time.Minute)
+	defer cancel()
+	err = wait.PollImmediateUntil(10*time.Second, func() (bool, error) {
```

Comment:
please can you add a logline above here?

Reply:
@jim-minter Done